Add Big Box promo cards

nwai90 opened the pull request:
(Only need to examine the last 3 commits here as everything else has been checked previously)

vincentneko commented on src/cards/promo/PharmacyUnion.ts:
```diff
@@ -0,0 +1,84 @@
+            player.megaCredits = Math.max(player.megaCredits - microbeTagCount * 4, 0)
+        }
+            
+        if (player.isCorporation(CorporationName.PHARMACY_UNION) && card.tags.includes(Tags.SCIENCE)) {
```
This will not work with Research card (2 sciences tags)
Mars University has the proper logic to handle this and can be used a source of inspiration

nwai90 replied:
👍 Great catch, will update

nwai90 replied:
@vincentneko Updated, can you see if this is ok?

alrusdi commented:
I have a question about that replacement cards. Are they playable now or we have a naming conflict?

nwai90 commented:
> I have a question about that replacement cards. Are they playable now or we have a naming conflict?

They should be playable, since they replace the original cards. Will test it for a bit to make sure that the 3 replacement cards can be played without issues

nwai90 commented:
I added missing HTML data for the 3 promo cards and fixed the incorrect name for Asteroid Deflection System

<img width="198" alt="Screenshot 2020-07-04 at 12 03 40 PM" src="https://user-images.githubusercontent.com/2408094/86504997-09a89580-bdf2-11ea-838c-dfa3494c0d5c.png">
<img width="199" alt="Screenshot 2020-07-04 at 11 52 42 AM" src="https://user-images.githubusercontent.com/2408094/86505000-0c0aef80-bdf2-11ea-819e-5b4a856970c1.png">
<img width="195" alt="Screenshot 2020-07-04 at 12 06 09 PM" src="https://user-images.githubusercontent.com/2408094/86505002-0dd4b300-bdf2-11ea-88d8-7dc2400b2d7a.png">
<img width="196" alt="Screenshot 2020-07-04 at 12 25 35 PM" src="https://user-images.githubusercontent.com/2408094/86505006-10370d00-bdf2-11ea-850f-f624f9e464dd.png">

nwai90 commented:
Also updated tile placement logic for Great Dam promo:

<img width="667" alt="Screenshot 2020-07-04 at 12 46 41 PM" src="https://user-images.githubusercontent.com/2408094/86505214-7e7ccf00-bdf4-11ea-9a2d-a311258f2ff1.png">

The tile has to be placed adjacent to an ocean.

nwai90 commented on src/styles/cards.less:
```diff
@@ -1607,6 +1607,12 @@ input[type=radio]:checked + .filterDiv {
 
+.card-beam-from-a-thorium-asteroid {
+    .title {
+      font-size: 13px;
```
Possibly automate this styling in future if needed, e.g. use `font-size: 14px` if title > 20 characters, and `font-size: 13px` if title > 24 characters
